### PR TITLE
Make the skip button clickable only once.

### PR DESCRIPTION
**Pull request by Nabile-Rahmani**
Addresses issue #1017.

### CODE REVIEW

File: osu.Game/Screens/Play/SkipButton.cs [hunk 1]
```diff
@@ -182,6 +182,7 @@ private class Button : OsuClickableContainer
             private FillFlowContainer flow;
             private Box background;
             private AspectContainer aspect;
+            private bool isClicked;
```
Comment (MrTheMake) [inline osu.Game/Screens/Play/SkipButton.cs]: I probably would disable the button via the `Enabled` variable instead of creating a new one.

Comment (Nabile-Rahmani) [inline osu.Game/Screens/Play/SkipButton.cs]: You're right.

File: osu.Game/Screens/Play/SkipButton.cs [hunk 2]
```diff
@@ -277,9 +277,16 @@ protected override bool OnMouseUp(InputState state, MouseUpEventArgs args)
 
             protected override bool OnClick(InputState state)
             {
+                if (!Enabled)
```
Comment (MrTheMake) [inline osu.Game/Screens/Play/SkipButton.cs]: Enabled.Value

Comment (Nabile-Rahmani) [inline osu.Game/Screens/Play/SkipButton.cs]: Implicit conversion. Looks nicer.

Comment (MrTheMake) [inline osu.Game/Screens/Play/SkipButton.cs]: Oh right, I see.

**Review by peppy — APPROVED** [review]

**Pull request merged**